src/App.js

```diff
@@ -19,55 +18,58 @@ import loader from './Assets/Images/loader.gif';
 import data from './Data/data-card-layout';
 import './App.css';
 
-
 class App extends Component {
   constructor() {
     super();
     this.state = { isLoading: true };
   }
 
   componentDidMount() {
+    /* questionable use of setTimeout
+    I would implement simply a call to setState
+    this.setState({ isLoading: true }); */
+
     setTimeout(() => this.setState({ isLoading: false }), 3000);
   }
 
   render() {
-    const { state: { isLoading } } = this;
+    const { isLoading } = this.state;
     if (isLoading) {
       return (
         <div className="container col-lg-6 col-md-4 col-sm-6 col-9 mx-auto ">
-          <img src={loader} className="loader" />
+          <img src={loader} className="_loader" alt="loading" />
         </div>
       );
     }
     return (
-      <div className="content-container">
-        <header>
+    <Router>
+        <Fragment>
           <Navigation.components.Navigation />
-        </header>
-        <Switch>
-          <Route
-            exact
-            path="/"
-            render={() => (
-              <div className="content">
-                <Introduction.components.Introduction />
```

Comment:
I've never seen or used that syntax before, also, I didn't remove those specifically. I did notice that someone attempted to use the same syntax in App.js with the Nav and Footer components, i.e. <Navigation.components.Navigation /> and <Footer.components.Footer />. I removed these specific modifications because it broke the site, at least locally, for me.

Reply:
but I don't think it breaks the site? that syntax is one of the design pattern, I implemented in app.js I encountered this pattern when I was working with carl zeiss AG in a project, and it is indeed useful when it comes to import reduxers, sagas all in index.js.

```
import * as actions from './actions';
import * as components from './Components';
import * as constants from './constants';
import * as containers from './Containers';
import reducer from './reducers';
import * as sagas from './sagas';
import * as selectors from './selectors'

export default {
  actions,
  components,
  containers,
  selectors,
  constants,
  reducer,
  sagas,
};

```

Reply:
My mistake!